tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/steps/PitclipseSteps.java

```diff
@@ -155,72 +178,52 @@ private void runPit(Runnable runnable) {
         }
     }
     
+    public void assertNoErrorsInWorkspace() {
+        Set<String> errors = errorsInWorkspace();
+        assertThat(errors, empty());
```

Comment:
Does the `empty()` matcher print the content of the set if it's not empty? I'd prefer to avoid a generic "was expecting an empty collection" error message and instead have the full list of the errors found in the workspace.

Reply:
Yes, it does, I tried that in a TDD way (I first made the test fail).
We could also add an additional informative error message if you want (the non empty set would still be printed on the screen)